{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Index\n",
    "\n",
    "pyOpenMS is an open-source Python library for mass spectrometry,\n",
    "specifically for the analysis of proteomics and metabolomics data in\n",
    "Python. pyOpenMS implements a set of Python bindings to the OpenMS\n",
    "library for computational mass spectrometry and is available for\n",
    "Windows, Linux and OSX.\n",
    "\n",
    "PyOpenMS provides functionality that is commonly used in computational\n",
    "mass spectrometry. The pyOpenMS package contains Python bindings for a\n",
    "large part of the [OpenMS](openms.de) library for mass spectrometry\n",
    "based proteomics. It thus provides access to a feature-rich, open-source\n",
    "algorithm library for mass-spectrometry based proteomics analysis.\n",
    "\n",
    "pyOpenMS facilitates the execution of common tasks in proteomics (and\n",
    "other fields of mass spectrometry) such as\n",
    "\n",
    "-   File handling (`mzXML`, `mzML`, TraML, mzTab, `FASTA`, pepxml,\n",
    "    protxml, mzIdentML among others)\n",
    "-   Chemistry (mass calculation, peptide fragmentation, isotopic\n",
    "    abundances)\n",
    "-   Signal processing (smoothing, filtering, de-isotoping, retention\n",
    "    time correction and peak-picking)\n",
    "-   Identification analysis (including peptide search, PTM analysis,\n",
    "    cross-linked analytes, FDR control, RNA oligonucleotide search and\n",
    "    small molecule search tools)\n",
    "-   Quantitative analysis (including label-free, metabolomics, `SILAC`,\n",
    "    `iTRAQ` and `SWATH`/DIA analysis tools)\n",
    "-   Chromatogram analysis (chromatographic peak picking, smoothing,\n",
    "    elution profiles and peak scoring for `SRM`/MRM/PRM/`SWATH`/DIA\n",
    "    data)\n",
    "-   Interaction with common tools in proteomics and metabolomics\n",
    "    -   Search engines such as Comet, Crux, Mascot, MSGFPlus, MSFragger,\n",
    "        Myrimatch, OMSSA, Sequest, SpectraST, XTandem\n",
    "    -   Post-processing tools such as percolator, MSStats, Fido\n",
    "    -   Metabolomics tools such as SIRIUS, CSI:FingerId\n",
    "\n",
    "background\n",
    "\n",
    "installation first_steps introduction\n",
    "\n",
    "ms_data chemistry peptides_proteins oligonucleotides_rna\n",
    "fragment_spectrum_generation spectrum_alignment digestion\n",
    "identification_data quantitative_data\n",
    "\n",
    "parameter_handling algorithms smoothing centroiding\n",
    "spectrum_normalization charge_isotope_deconvolution feature_detection\n",
    "map_alignment feature_linking peptide_search chromatographic_analysis\n",
    "quality_control mass_decomposition export_files_GNPS\n",
    "\n",
    "identification_accurate_mass untargeted_metabolomics_preprocessing\n",
    "\n",
    "contribute\n",
    "\n",
    "reading_raw_ms_data other_ms_data_formats mzml_files\n",
    "scoring_spectra_hyperscore export_pandas_dataframe\n",
    "query_msexperiment_massql memory_management pyopenms_in_r\n",
    "build_from_source wrapping_workflows_new_classes interactive_plots\n",
    "interfacing_ml_libraries\n",
    "\n",
    "apidoc\n",
    "\n",
    "support faq glossary\n",
    "\n",
    "# Indices and Tables\n",
    "\n",
    "-   `genindex`"
   ],
   "id": "8812df3e-0635-4b05-895c-fe079ebbcb0e"
  }
 ],
 "nbformat": 4,
 "nbformat_minor": 5,
 "metadata": {}
}
